src/Rendering/Images/createImageRenderingActor.js

```diff
@@ -175,7 +219,10 @@ const eventResponses = {
   // updateRenderedImage->applyRenderedImage->updateCroppingParametersFromImage->CROPPING_PLANES_CHANGED
   // because image size may change across scales.
   CROPPING_PLANES_CHANGED_BY_USER: {
-    target: 'imageBoundsDebouncing',
+    target: 'debouncingUpdatingImage',
+  },
+  CAMERA_MODIFIED: {
+    target: 'debouncingUpdatingImage',
```

Comment:
debouncingUpdatingImage -> debouncedImageUpdate (remove passive voice)

Reply:
Was going with a convention I read that state are "points in time" rather than functions, so have been throwing a bunch of `ing` endings around.

Reply:
Ah, to satisify the "points in time" and active voice desirables, how about `imageUpdateDebounced`?

Is it possible to not have this state at all and get the functionality with a [condition](https://xstate.js.org/docs/guides/guards.html)?

Reply:
How would the condition work?  Something like
```
CAMERA_MODIFIED: {
  actions: [assignDebounceTime, c => setTimeout(() => c.server.send('UPDATE_RENDERED_IMAGE_DEBOUCE'), 500)]
  }
UPDATE_RENDERED_IMAGE_DEBOUCE: {
    cond: isAfterDebouceTime, 
    target: 'updatingImage',
}
```
Seems like the declarative nature of a state is preferred over messing with the context.

Reply:
Yes, that may be roughly be how to implement it, or maybe a boolean instead of a time in the context?

We do want a declarative state, and  we also want it to be valid in its serialized form. If we stop the machine, serialize it, restart it, will debouncing related entries be valid?